logprep/util/defaults.py

```diff
@@ -1,5 +1,18 @@
 """Default values for logprep."""
 
+from enum import Enum
+
+
+class EXITCODES(Enum):
+    """Exit codes for logprep."""
+
+    SUCCESS = 0
+    ERROR = 1
+    CONFIGURATION_ERROR = 2
+    PIPELINE_ERROR = 3
```

Comment:
That's something we could add to the documentation. Every Enum-Entry could get a doc string explaining when these errors could happen and then you could integrate it to the shpinx doc somewhere.

Reply:
done :D